images/cst/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# syntax=docker/dockerfile:1.1-experimental
+
+# Copyright 2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG GOLANG_IMAGE=golang:1.14.3@sha256:1e36f8e9ac49d5ee6d72e969382a698614551a59f4533d5d61590e3deeb543a7
```

Comment:
Any reason not to use `golang:1.14.4` here?

Reply:
It's just not been updated yet, as this same image is used in a few other places, I'd rather not update the Go version in this PR.

Reply:
Ah, didn't realize that this was copied from upstream. LGTM in that case.